src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -2002,6 +2060,11 @@ void ValidateHelmInstall(string helmPath, string featureName)
             {
                 throw new Exception("Invalid configuration options BatchNodesSubnetId and BatchSubnetName are mutually exclusive.");
             }
+
+            if (!string.IsNullOrWhiteSpace(configuration.AksClusterName) && !configuration.UsePostgreSqlSingleServer)
+            {
+                throw new Exception("If providing an existing AKS cluster, --UsePostgreSqlSingleServer must be set to true.");
```

Comment:
Isn't single server being retired in 19 months?

Reply:
We'll see, you still can't create private endpoint to a flexible postgres server. So if postgres is in our vnet and our services are running in AKS in another vnet, there's no good way to connect. We'd have to set postgres to public, or peer the vnets both of which are aren't ideal.